<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iLand Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon4.png</normaloff>:/icon4.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="PaintArea" name="PaintWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOutput_menu">
    <property name="title">
     <string>Output-menu</string>
    </property>
    <addaction name="actionTreelist"/>
    <addaction name="actionFON_grid"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDebug_Data">
    <property name="title">
     <string>Debug Data</string>
    </property>
    <addaction name="actionSelect_Data_Types"/>
    <addaction name="actionTree_NPP"/>
    <addaction name="actionTree_Partition"/>
    <addaction name="actionTree_Growth"/>
   </widget>
   <addaction name="menuOutput_menu"/>
   <addaction name="menuView"/>
   <addaction name="menuDebug_Data"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEdit_XML_settings"/>
   <addaction name="separator"/>
   <addaction name="actionModelCreate"/>
   <addaction name="actionModelRun"/>
   <addaction name="separator"/>
   <addaction name="actionModelDestroy"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockEditor">
   <property name="windowTitle">
    <string>DockDock</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="editStack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="pageXML">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="initFileName">
             <property name="text">
              <string>fon.xml</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="openFile">
             <property name="text">
              <string>Open</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveFile">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="applyXML">
             <property name="text">
              <string>Apply!</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="iniEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>1000</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageFON">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Stamping</string>
            </attribute>
            <widget class="QPushButton" name="stampTrees">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>10</y>
               <width>75</width>
               <height>23</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>stamp trees</string>
             </property>
            </widget>
            <widget class="QPushButton" name="fonRun">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>75</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Setup FON</string>
             </property>
            </widget>
            <widget class="QGroupBox" name="gEditTree_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>331</width>
               <height>111</height>
              </rect>
             </property>
             <property name="title">
              <string>Edit tree</string>
             </property>
             <widget class="QDoubleSpinBox" name="treeDbh">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>30</y>
                <width>62</width>
                <height>22</height>
               </rect>
              </property>
              <property name="toolTip">
               <string>diameter (cm)</string>
              </property>
              <property name="maximum">
               <double>299.990000000000009</double>
              </property>
             </widget>
             <widget class="QDoubleSpinBox" name="treeHeight">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>30</y>
                <width>62</width>
                <height>22</height>
               </rect>
              </property>
              <property name="toolTip">
               <string>Height [m]</string>
              </property>
             </widget>
             <widget class="QPushButton" name="treeChange">
              <property name="geometry">
               <rect>
                <x>220</x>
                <y>10</y>
                <width>75</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>Change</string>
              </property>
             </widget>
             <widget class="QDoubleSpinBox" name="treePosX">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>60</y>
                <width>62</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
             <widget class="QDoubleSpinBox" name="treePosY">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>60</y>
                <width>62</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="treeImpact">
              <property name="geometry">
               <rect>
                <x>220</x>
                <y>60</y>
                <width>91</width>
                <height>41</height>
               </rect>
              </property>
              <property name="text">
               <string>(value)</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pbSetAsDebug">
              <property name="geometry">
               <rect>
                <x>220</x>
                <y>40</y>
                <width>75</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>debug tree</string>
              </property>
             </widget>
            </widget>
            <widget class="QPushButton" name="pbResults">
             <property name="geometry">
              <rect>
               <x>270</x>
               <y>10</y>
               <width>71</width>
               <height>23</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>copies tree results in clipboard.</string>
             </property>
             <property name="text">
              <string>Results</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockLogviewer">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Log output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="logOutput">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Visualization Options</string>
       </property>
       <widget class="QRadioButton" name="visFon">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>84</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>FON</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="visImpact">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Impact values</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="visDomGrid">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>101</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>dominace grid</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="visAutoScale">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Autoscale colors</string>
        </property>
       </widget>
       <widget class="QLabel" name="fonValue">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>(value)</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="visDomHeight">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>51</width>
          <height>18</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>for vis. of influence grid: when checked, the color shows the FON-value if scaled to the height in combo-box right, and not for the value of the actual height-dominance-grid.</string>
        </property>
        <property name="text">
         <string>scale</string>
        </property>
       </widget>
       <widget class="QComboBox" name="visScaleValue">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>110</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;for vis. of influence grid: height value in meters.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>80</string>
         </property>
        </item>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionEdit_XML_settings">
   <property name="icon">
    <iconset>
     <normaloff>:/source_xml.png</normaloff>:/source_xml.png</iconset>
   </property>
   <property name="text">
    <string>XML settings</string>
   </property>
  </action>
  <action name="actionTreelist">
   <property name="text">
    <string>treelist</string>
   </property>
  </action>
  <action name="actionFON_grid">
   <property name="text">
    <string>FON grid</string>
   </property>
  </action>
  <action name="actionModelCreate">
   <property name="icon">
    <iconset>
     <normaloff>:/www.png</normaloff>:/www.png</iconset>
   </property>
   <property name="text">
    <string>Create Model</string>
   </property>
  </action>
  <action name="actionModelDestroy">
   <property name="icon">
    <iconset>
     <normaloff>:/user-trash.png</normaloff>:/user-trash.png</iconset>
   </property>
   <property name="text">
    <string>Destroy</string>
   </property>
  </action>
  <action name="actionModelRun">
   <property name="icon">
    <iconset>
     <normaloff>:/go-next.png</normaloff>:/go-next.png</iconset>
   </property>
   <property name="text">
    <string>Run Model</string>
   </property>
  </action>
  <action name="actionTree_Partition">
   <property name="text">
    <string>Tree Partition</string>
   </property>
  </action>
  <action name="actionTree_Growth">
   <property name="text">
    <string>Tree Growth</string>
   </property>
  </action>
  <action name="actionSelect_Data_Types">
   <property name="text">
    <string>Select Data Types</string>
   </property>
  </action>
  <action name="actionTree_NPP">
   <property name="text">
    <string>Tree NPP</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintArea</class>
   <extends>QWidget</extends>
   <header>paintarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
